x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -32,14 +32,21 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
         onChange={value => dispatch({ field: 'description', value })}
         value={formFields.description.value}
       />
-      {/*
       <EditTransformFlyoutFormTextInput
-        defaultValue={config.dest.index}
-        label={i18n.translate('xpack.transform.transformList.editFlyoutFormDestinationIndexLabel', {
-          defaultMessage: 'Destination Index',
+        errorMessages={formFields.docsPerSecond.errorMessages}
+        helpText={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormDocsPerSecondHelptext',
+          {
+            defaultMessage:
+              'To enable throttling, set a limit of documents per second of input documents.',
```

Comment:
One for a follow-up - is there any guidance we can give as to what is a sensible value here?